### PR TITLE
Docs: Add Team support tab + subpages 

### PR DESCRIPTION
**Pull request by cintiapub**
### Summary
I am adding a new tab "Team support" with new content and pages, and doing some content adjustments.

#### What changed?

- 1 new tab "Team support" + 5 new pages under this new tab
- Deleting the page "How to work with us". The new page "Contributions" (under Team support) will replace this page. 
- Changing the link on the Home page: "How to work with us" to "Contributions"

#### Why?

It will support comprehension and reduce long scrolls to find relevant information regarding team support. 
In addition, we're breaking the content into different pages to improve readability, and adding a new page:"**Request a component**." It will help us to track component/updates to existent components requests through Jira. 

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-5799)
- [Coda](https://coda.io/d/Team-support-pages_dsH4qyoeNDn/Team-support_suhaY#_luPC_)
- [Figma](https://www.figma.com/file/KKKHtAgSzA1ZyBX7zGTtny/Team-support?node-id=0%3A1&t=YRe7GMYv5iuD4Em6-1)

### Checklist

- [ ] Added unit and Flow Tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

**Comment by AlbertCarreras** [comment]
Screenshot by Dropbox Capture


Minor thingy!

Just CMD SHIFT P and select Format Document (Forced)

File: docs/markdown/team_support/component_request.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    Will this unlock additional functionality that other teams need?
+
+We prioritize work following the same cycles as product teams within Pinterest. Knowing other teams' needs before prioritization starts will help inform our roadmap. If we do not have the capacity for the work but believe we should add it to Gestalt, we may ask if a product team can continue the work. Otherwise, it will go into our backlog.
+
+Our team will update your ticket request following our prioritization standards, and you will be able to follow the progress of your ticket. 
+
+<br/>
+
+### Overal timeline
```
Comment (AlbertCarreras) [inline docs/markdown/team_support/component_request.md:52]: typo Overall

File: docs/markdown/team_support/component_request.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+<br/>
+<br/>
+
+<br/>
+<br/>
+
+<iframe style={{border:0}} width="100%" height="1000" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FKKKHtAgSzA1ZyBX7zGTtny%2FTeam-support%3Fnode-id%3D1233%253A1931%26t%3DYRe7GMYv5iuD4Em6-1" allowFullScreen></iframe>
+
+For Imagery, logos, collage, illustrations and brand graphics, visit [Brand resources](https://brand.pinterest.com/resources). Gestalt doesn't provide brand assets.
```
Comment (AlbertCarreras) [inline docs/markdown/team_support/component_request.md:25]: typo imagery

File: docs/markdown/team_support/contributions.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Making a design contribuition
+
+Design system is about collaboration and building blocks! Gestalt offers the elements you need to create a consistent product. If you have an idea to improve Pinterest surfaces by partnering with the Gestalt team, please follow the steps below. We would love to collaborate with you!Follow the steps below to make a design contribution: 
+
+1. **Present your idea or suggestion during our Office Hour meetings**
+    [Sign up](https://pinch.pinadmin.com/gestaltSignUp) for an office hours slot with your discussion topic! 
+    EBring your work early-on— What we build into Gestalt comes from the teams across Pinterest, so we’d love to provide guidance and support early and often! The earlier we see the work, the better we're able to plan and the higher the likelihood we'll be able to help. It does not need to be a polished presentation but should have enough detail to help us understand the help you need and provide support.
```
Comment (AlbertCarreras) [inline docs/markdown/team_support/contributions.md:15]: typo Bring

**Comment by rlingineni** [comment]
@cintiapub  - can you run `yarn format` in your vscode terminal, and then recommit. Looks like one of the formatting checks is failing here

**Comment by rlingineni** [comment]
ooh - cintia, I forgot one last step.

It's failing on missing accessibility tests for the pages. Promise we're almost there. It may seem a little complex, but I think you should be able to totally do it:

Error: ❌ The following doc pages do not have an accessibility integration test: overview_team support,be a gestalt advocate,component request,contributions,get help,training

Do you think you can navigate to this folder in [VSCode](https://github.com/pinterest/gestalt/tree/master/playwright/accessibility), it's an accessiblity folder.

Then make the following files?
```
overview_team support.spec.mjs
be a gestalt advocate.spec.mjs
component request.spec.mjs
contributions.spec.mjs
get help.spec.mjs
training.spec.mjs
```

You can look at what goes into the files, by clicking on the ones next to it, and copy pasting.

You will change two things on this file
```
// @flow strict
import { test } from '@playwright/test';
import expectAccessiblePage from './expectAccessiblePage.mjs';

// change the test NAME
test('Team support overview Accessibility check', async ({ page }) => {
  await page.goto('/team_support/overview');   // update the website url
  await expectAccessiblePage({ page });
});

```


After you're done making all the changes, in the other terminal, run `yarn test`. If everything works, at the end of the run, it'll give you a checkmark, so you'll know if you did it correctly. Happy to help tomorrow

**Review by rlingineni — APPROVED** [review]
Looking good!

**Pull request merged**